packages/itwinui-css/src/input-container/input-container.scss

```diff
@@ -104,6 +104,14 @@
     &:is(:not(:first-child)) {
       padding-inline-start: var(--iui-size-2xs);
     }
+
+    &::placeholder {
+      --_iui-input-text-color: var(--iui-color-text-placeholder);
```

Comment:
it looks like this variable only gets used in the parent, so it doesn't need to be set here. we can directly use `color: var(--iui-color-text-placeholder)`

Reply:
remove line 113 
<img width="499" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/ab3a24d7-4f55-48bb-904e-e5d74e48ea2e">

Reply:
Updated?